### PR TITLE
add change for new version

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1012
https://github.com/open-orchestra/open-orchestra-model-interface/pull/106
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/404

File: BackofficeBundle/Manager/NodeManager.php [hunk 1]
```diff
@@ -56,6 +60,7 @@ public function __construct(
     )
     {
         $this->nodeManager = $nodeManager;
```
Comment (nicolasThal) [inline BackofficeBundle/Manager/NodeManager.php]: the model nodeManager is not used anymore

**Comment by itkg-rboessel** [comment]
@nicolasThal done

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**